metrics.ini

```diff
@@ -1624,6 +1624,23 @@ MusicWheelSortMessageCommand=%function(self, params) \
 	self:ChangeSort('SortOrder_'..params.order) \
 end
 
+[ScreenPracticeMenu]

```

Comment:
What's the difference between ScreenPracticeMenu and ScreenPractice? And what is PracticeMenu?

Reply:
Ah, on closer inspection "ScreenPracticeMenu" isn't needed, it was originally cribbed from the "EditMode" equivalents, where this was essentially the song select. "EditMenu" I think also supports that use case.

That is to say, I'll get them removed, "ScreenPractice" is probably the only one needed here.